### PR TITLE
Add bin_size_list as an optional input with a default of 10000

### PR DESCRIPTION
**Pull request by sophiacrennan**

### CODE REVIEW

File: pipelines/skylab/snap-atac/README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ The pipeline inputs are detailed in the table below. You can test the workflow b
 | input_reference | File | Reference bundle that is generated with bwa-mk-index-wdl found [here](https://github.com/HumanCellAtlas/skylab/blob/master/library/accessory_workflows/build_bwa_reference/bwa-mk-index.wdl)|
 | genome_name | String | Name of the genomic reference (name that precedes the “.tar” in the input_reference) |
 | output_bam  | String  | Name for the output BAM |
+| bin_size_list  | String  | List of bin sizes used to generate cell-by-bin matrices; default is 10000 bp |
```
Comment (ekiernan) [inline pipelines/skylab/snap-atac/README.md:65]: Can users put in more than one number here? So they could do something like`10000 5000`?

Comment (sophiacrennan) [reply pipelines/skylab/snap-atac/README.md:65]: Thats a good question. @khajoue2 do you know?

File: pipelines/skylab/scATAC/ATAC.wdl [hunk 2]
```diff
@@ -635,7 +638,7 @@ task SnapCellByBin {
 
   parameter_meta {
     snap_input: "the bam to passed into snaptools tools"
-    bin_size_list: "space separated list of bins to generate"
+    bin_size_list: "list of bins to generate"
```
Comment (ekiernan) [inline pipelines/skylab/scATAC/ATAC.wdl:641]: Might want to update if it's not an actual list- "Bin size value"

File: pipelines/skylab/snap-atac/scATAC.wdl [hunk 3]
```diff
@@ -19,6 +20,7 @@ workflow scATAC {
         input_reference: "tar file with BWA reference, generated with the build_bwa_reference pipeline"
         output_bam: "output BAM file name"
         genome_name: "name of the genome for snap atac"
+        bin_size_list: "list of bin sizes to generate"
```
Comment (ekiernan) [inline pipelines/skylab/snap-atac/scATAC.wdl:23]: Maybe define as "Bin size value"

File: pipelines/skylab/scATAC/ATAC.wdl [hunk 4]
```diff
@@ -50,6 +52,7 @@ workflow ATAC {
     min_map_quality: "the minimum mapping quality to be filtered by samtools view and snap-pre (snaptools task)"
     max_fragment_length: "the maximum fragment length for filtering out reads by gatk and snap-pre (snaptools task)"
     output_base_name: "base name to be used for the pipelines output and intermediate files"
+    bin_size_list: "list of bin sizes to generate"
```
Comment (ekiernan) [inline pipelines/skylab/scATAC/ATAC.wdl:55]: Update to "Bin size value"

File: pipelines/skylab/snap-atac/scATAC.wdl [hunk 5]
```diff
@@ -193,7 +195,7 @@ task SnapCellByBin {
 
     parameter_meta {
        snap_input: "input snap file to generate bins for"
-       bin_size_list: "list of bin sizes to generate"
+       bin_size_value: "space separated list of bins to generate"
```
Comment (ekiernan) [inline pipelines/skylab/snap-atac/scATAC.wdl:198]: Still says bin_size_value

File: pipelines/skylab/scATAC/ATAC.wdl [hunk 6]
```diff
@@ -635,7 +638,7 @@ task SnapCellByBin {
 
   parameter_meta {
     snap_input: "the bam to passed into snaptools tools"
-    bin_size_list: "space separated list of bins to generate"
+    bin_size_value: "space separated list of bins to generate"
```
Comment (ekiernan) [inline pipelines/skylab/scATAC/ATAC.wdl:641]: Still says bin_size_value

**Pull request merged**